reference/src/layout/scalars.md

```diff
@@ -22,7 +23,7 @@ taking on the values 1 (`true`) or 0 (`false`).
 ## `char`
 
 Rust char is 32-bit wide and represents an [unicode scalar value]. The alignment
-of `char` is _implementation-defined_.
+of `char` is _implementation-defined_ (Citation needed).
```

Comment:
I wonder that whether `char` alignment is really implementation defined.

Reply:
Well is it explicitly specified anywhere?

Reply:
It doesn't. But I wonder can we define this information in the Reference book?
And/Or explain why char is implementation defined.

Reply:
Hi, I add citation to the Rust reference.